pkgs/applications/graphics/drawpile/default.nix

```diff
@@ -69,23 +69,15 @@ let
 
 in mkDerivation rec {
   pname = "drawpile";
-  version = "2.1.17";
+  version = "2.1.19";
 
   src = fetchFromGitHub {
     owner = "drawpile";
     repo = "drawpile";
     rev = version;
-    sha256 = "sha256-AFFY+FcY9ExAur13OoWR9285RZtBe6jnRIrwi5raiCM=";
+    sha256 = "sha256-MNmzcqTHfMms6q3ZilrChE5WoGzGxnAOkB0a75udA1I=";
   };
 
-  patches = [
-    # fix for libmicrohttpd 0.9.71
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Done
9c0644262fc9b5a7c52c25b74d919c324fb34b6a

Reply:
Thanks! By the way, do we have a linter I can use to check for this kind of easily forgettable stuff?

Reply:
https://github.com/jtojnar/nixpkgs-hammering